### PR TITLE
docs: add a guide on how to clean up action's logs (close #4935)

### LINKED ISSUE #4935: docs: document action logs cleanup
Document the process of cleaning up action logs like there exists for [cleaning up event triggers logs](https://hasura.io/docs/1.0/graphql/manual/event-triggers/clean-up.html). 

The documentation should also include the possible side-effects of cleaning up an action log. For example, clearing an action log which has an active subscription running on it.

### PR DESCRIPTION
**Pull request by codeliezel**
### Description
This PR adds a guide on how to clean up action's logs.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components
- [x] Docs

### Related Issues
https://github.com/hasura/graphql-engine/issues/4935

### Affected pages
https://deploy-preview-6163--hasura-docs.netlify.app/graphql/core/actions/clean-up-action-logs.html

### CODE REVIEW

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Actions has a table managed by Hasura:
+
+- ``hdb_catalog.hdb_action_log``: This is the table that stores all captured logs.
+
+Option 1: Get all logs
+----------------------
+.. code-block:: SQL
+   
+   SELECT * FROM hdb_catalog.hdb_action_log;
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:31]: For these statements we need some guidance on how to run these. We should introduce tabs also to show console / cli / api workflow? You can check the other actions pages for guidance.

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Option 2: Delete one log
+------------------------
+.. code-block:: SQL
+   
+   DELETE FROM hdb_catalog.hdb_action_log WHERE id = '<action-id>';
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:37]: console / cli / api workflow

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Option 3: Clear all logs
+------------------------
+.. admonition:: Warning
+ 
+   This will clear all captured logs.
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:43]: This we already say in the title. If the action is irreversible, we can do a note saying "Clearing all logs is irreversible". But before we should check if this is actually the case.

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+.. code-block:: SQL
+
+   DELETE FROM hdb_catalog.hdb_action_log;
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:47]: console / cli / api workflow

**Review by marionschleifer — CHANGES_REQUESTED** [review]
@codeliezel I left some comments 🙂

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+Hasura stores actions logs from asynchronous operations. If there are a lot of logs, the tables can get huge and you may want to prune them.
+You can use any of the following options to prune your logs depending on your need.
+
+The table involved
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:20]:
```suggestion
Table involved
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 6]
```diff
@@ -0,0 +1,47 @@
+.. meta::
+   :description: clean up actions logs
+   :keywords: hasura, docs, actions, clean up, async actions
+
+.. _clean_up_actions_logs:
+
+Clean up actions logs
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:7]:
```suggestion
Clean up action logs
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 7]
```diff
@@ -0,0 +1,47 @@
+.. meta::
+   :description: clean up actions logs
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:2]:
```suggestion
   :description: clean up action logs
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+=====================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+Hasura stores actions logs from asynchronous operations. If there are a lot of logs, the tables can get huge and you may want to prune them.
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:17]:
```suggestion
Hasura stores action logs of asynchronous operations. As the table gets larger, you may want to prune them.
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+You can use any of the following options to prune your logs depending on your need.
+
+The table involved
+------------------
+
+Actions has a table managed by Hasura:
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:23]:
```suggestion
There is a specific table for action logs that is managed by Hasura:
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+- ``hdb_catalog.hdb_action_log``: This is the table that stores all captured logs.
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:25]:
```suggestion
- ``hdb_catalog.hdb_action_log``: This table stores all captured action logs.
```

File: docs/graphql/core/actions/clean-up-actions-logs.rst (hunk 3 above)
Comment (marionschleifer) [reply docs/graphql/core/actions/clean-up-actions-logs.rst:43]: Also, we can add this note after the command

File: docs/graphql/core/actions/clean-up-actions-logs.rst [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+   
+   SELECT * FROM hdb_catalog.hdb_action_log;
+
+Option 2: Delete one log
+------------------------
+.. code-block:: SQL
+   
+   DELETE FROM hdb_catalog.hdb_action_log WHERE id = '<action-id>';
+
+Option 3: Clear all logs
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-actions-logs.rst:39]: Let's use either `Clear` or `Delete` consistently throughout the page

File: docs/graphql/core/actions/clean-up-actions-logs.rst (hunk 3 above)
Comment (codeliezel) [reply docs/graphql/core/actions/clean-up-actions-logs.rst:43]: Yes, it is irreversible, so I will change the statement. @marionschleifer

**Review by marionschleifer — CHANGES_REQUESTED** [review]
@codeliezel this looks good 🙂 I added some small suggestions :)

File: docs/graphql/core/actions/clean-up-actions-logs.rst (hunk 6 above)
Comment (codeliezel) [reply docs/graphql/core/actions/clean-up-actions-logs.rst:7]: Not necessary :)

File: docs/graphql/core/actions/clean-up-action-logs.rst [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+Option 3: Delete all logs
+-------------------------
+.. code-block:: SQL
+
+   DELETE FROM hdb_catalog.hdb_action_log;
+
+.. admonition:: Warning
+ 
+   Deleting all logs is irreversible.
```
Comment (marionschleifer) [inline docs/graphql/core/actions/clean-up-action-logs.rst:48]:
```suggestion
   Deleting logs is irreversible, so be careful with this action.
```

Comment (marionschleifer) [reply docs/graphql/core/actions/clean-up-action-logs.rst:48]: Let's say this more general. Deleting one log is also irreversible from what I understand 🙂

Comment (codeliezel) [reply docs/graphql/core/actions/clean-up-action-logs.rst:48]: Yes, that is true @marionschleifer

**Review by marionschleifer — APPROVED** [review]

**Pull request closed without merging**